test/index.spec.js

```diff
@@ -0,0 +1,21 @@
+import { createLocalVue, shallowMount, Component } from '@vue/test-utils'
+import VueAnime from '../index.js'
+import Vue from 'vue';
+
+describe('VueAnime', () => {
+  it('should export the expected properties on the component', (done) => {
+    const testComp = 'hello-world';
+    const testCompBody = {
+      template: '<div></div>',
+      mounted() {
+        expect(typeof this.$anime).to.equal('function');
+        done();
+      }
+    };
+    const localVue = createLocalVue();
+
+    localVue.use(VueAnime)
+    Vue.component(testComp, testCompBody);
+    const wrapper = shallowMount(localVue.component(testComp), { localVue });
+  })
+})
```

Comment:
For best practice
[why-should-text-files-end-with-a-newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)
[no-newline-at-end-of-file](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)

Reply:
You're not wrong, I think my IDE used to do this automatically but I guess it doesn't anymore. Good catch.